sonar-plugin-api/src/main/java/org/sonar/api/server/rule/RuleDescriptionSection.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.api.server.rule;
+
+public interface RuleDescriptionSection {
+
+  String getKey();
+
+  String getHtmlDescription();
```

Comment:
If we don't plan to let users set a description with Markdown, it might be a good time to deprecate markdown descriptions everywhere

Reply:
It seems to be indeed a good opportunity to deprecate the support for markdown. If other agrees, I'll go on with the change 👍

Reply:
Issue https://jira.sonarsource.com/browse/PLUGINAPI-5 opened :)